2-projects/1-todo-list/script.js

```diff
@@ -17,6 +45,10 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  let inputEl = document.getElementById("todoInput");
+  todos.push({ task: inputEl.value, completed: false });
+
+  populateTodoList(todos);
```

Comment:
your `addNewTodo` function works really well - good job on using the same object format for your new todo item too. Remember to clear your inputEl after an item has been to the todo list